src/legacy/core_plugins/kibana/public/discover/doc_table/components/table_header/table_header.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { IndexPatternEnhanced } from 'ui/index_patterns/_index_pattern';
+// @ts-ignore
+import { shortenDottedString } from '../../../../../common/utils/shorten_dotted_string';
+import { TableHeaderColumn } from './table_header_column';
+import { SortOrder, getDisplayedColumns } from './helpers';
+
+interface Props {
+  columns: string[];
+  hideTimeColumn: boolean;
+  indexPattern: IndexPatternEnhanced;
+  isShortDots: boolean;
+  onChangeSortOrder: (name: string, direction: 'asc' | 'desc') => void;
+  onMoveColumn: (name: string, index: number) => void;
+  onRemoveColumn: (name: string) => void;
+  sortOrder: SortOrder;
+}
+
+export function TableHeader({
+  columns,
+  hideTimeColumn,
+  indexPattern,
+  isShortDots,
+  onChangeSortOrder,
+  onMoveColumn,
+  onRemoveColumn,
+  sortOrder,
+}: Props) {
+  const displayedColumns = getDisplayedColumns(columns, indexPattern, hideTimeColumn, isShortDots);
+  const [currColumnName, currDirection = 'asc'] = sortOrder;
+
+  return (
+    <tr data-test-subj="docTableHeader" className="kbnDocTableHeader">
+      <th style={{ width: '24px' }}></th>
```

Comment:
In the original angular template this was `1%`, of course this should be removed generally when the whole table is reactified + EUItified

Reply:
Should we add a `TODO` or a ticket here so future people know what's up with this (even though there's a good chance the future people are you, seems like good practice)

Reply:
This won't survive EUIfication, which will follow. It's just to keep the old way for a while, a bit more improved. There's no way this is kept, so I think in this case it's ok not to add a TODO here.